### ISSUE TITLE
Set up a process for adding existing issues from arch-bom owned repos to project

### ISSUE THREAD

**Issue by timmc-edx** [edx/edx-arch-experiments#497]
In https://github.com/edx/edx-arch-experiments/issues/494 we're ensuring that new issues in our 30-odd repos are added to the Arch-BOM GH project. We should also ensure that existing issues are added as well.

Possible process:

1. Set up a wiki listing our repos, annotated with their status
1. The on-call person works through the list of repos, closing out any tickets that can obviously be closed out.
    - This can be a low-priority task.
    - The on-call person works through the tickets from low ID to high.
    - The wiki page is updated with the highest number that on-call has gotten to.
1. The team also works through the list, following on-call or possibly overtaking them.
    - This should be done any time we run out of tickets on our board to prioritize.
    - The team prioritizes the tickets, maybe closing or updating them.
    - Again, the wiki page is updated with progress indicators.

(Possible other ticket: Ensuring that any repos that are newly under our ownership also have their issues added.)

**Comment by robrap** [comment]
@timmc-edx: Is this done, or done enough?

**Comment by timmc-edx** [comment]
I don't think we ever developed a process for dealing with existing tickets. (Or my memory is failing me.)

**Comment by robrap** [comment]
I see. I was getting confused. We should note that the automation ticket for new issues was closed on Nov 7, 2023, so this ticket applies to issues from before that date (give or take a few days).

**Comment by jristau1984** [comment]
This can be solved by simply searching each owned repo for issues with no associated project.  Once everything that comes up should be attached to this project, there is nothing left to consider.

**Comment by robrap** [comment]
@jristau1984: I think this was completed when we were in github. They all landed in our github project as No Status tickets. The question will be what process do we want now that we are in Jira? We could continue to look at the github board to see if any new tickets arrive. If we stop looking at edx-platform tickets, we won't have warning about relevant changes, but it would also cut down any triage. Thoughts?
